<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>forgetPassword</class>
 <widget class="QDialog" name="forgetPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* QLineEdit 样式 */
QLineEdit {
    color: white;
       font-family: &quot;Microsoft YaHei&quot;;
	background-color: lightblue;
    font-size: 28px; /* 提高字体大小 */
    border-style: outset;
    border-radius: 10px;

border: 2px solid #87CEEB; /* 增加边框以突出输入框 */
}


/* QLabel 样式 */
QLabel {
 font-family: &quot;Microsoft YaHei&quot;;
	background: rgba(85, 170, 255, 0);
    color: black; /* 黑色字体 */
    font-size: 28px; /* 字体大小 */
    padding: 2px; /* 内边距 */

    text-align: center; /* 居中对齐 */
}













</string>
  </property>
  <widget class="QLineEdit" name="phoneNumberLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="userNameLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="forgetButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>420</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 28px;
    color: white;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                  stop:0 #FDC245, stop:1 #FB9E23);
border-radius: 10px;

}
QPushButton:hover {
background-color:#FAC85C;
}</string>
   </property>
   <property name="text">
    <string>找回密码</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>110</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="register_userName_3">
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="register_userSex_3">
      <property name="text">
       <string>手机号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="register_userPhoneNumber_3">
      <property name="text">
       <string>验证码</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="checkLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="getCheckButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>290</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 20px;
    color: white;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                  stop:0 #FDC245, stop:1 #FB9E23);
border-radius: 10px;

}
QPushButton:hover {
background-color:#FAC85C;
}
</string>
   </property>
   <property name="text">
    <string>获取</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
